grommet-leaflet/src/layers/Controls.jsx

```diff
@@ -33,7 +33,13 @@ const Controls = ({ locations }) => {
 
   // on mount, zoom to the bounds of the locations
   if (bounds) {
-    map.flyToBounds(bounds, { duration: flyToBoundsDuration });
+    if (
+      JSON.stringify(bounds._northEast) === JSON.stringify(bounds._southWest)
+    ) {
+      // if the bounds are the same, zoom to the bounds of the locations
```

Comment:
There's nothing in this `if` block. Is that intentional?

If not can we rewrite the logic to avoid the empty block?

Reply:
This needs to be solved for.

Currently the application crashes if the `flyToBounds` function is called where the latlng param is called with the northEast and southWest bounds being equivalent. In this case, we still want to zoom and fit the data appropriately, but need to modify the approach.

Reply:
Gotcha, thanks for clarifying.